target/bin/restrict-access

```diff
@@ -19,20 +19,20 @@ case ${2} in
     ;;
   *)
     usage
-    _exit_with_error "missing parameters. Specify \"send\" or \"receive\""
+    _exit_with_error "Missing parameters. Specify 'send' or 'receive'"
     ;;
 esac
```

Comment:
The cases here were not converted to the style you've changed to further down in the file.

Reply:
👍🏼

Reply:
Done in 08a05debd67e69c3e5010be547c0cd0ca65a28c7